components/centraldashboard/public/components/utilities-mixin_test.js

```diff
@@ -1,37 +1,117 @@
+/* eslint-disable max-len */
```

Comment:
Any particular reason for this? I'd suggest removing it and trimming the lengthy lines to be under the 80 character limit.

Reply:
Removed, I think I copied this over from other tests.